source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=====================================
+Set maxTimeMS for Aggregation Queries
+=====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the ``maxTimeMS`` parameter on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|.
+
+About This Task 
```

Comment:
[question]

Is there a default maxTimeMS value we should mention here?

Reply:
Added!